spring-integration-core/src/main/java/org/springframework/integration/store/AbstractKeyValueMessageStore.java

```diff
@@ -206,7 +205,7 @@ public MessageGroupMetadata getGroupMetadata(Object groupId) {
 	}
 
 	@Override
-	public void addMessagesToGroup(Object groupId, Message<?>... messages) {
+	public void addMessagesToGroupInner(Object groupId, Message<?>... messages) {
```

Comment:
We prefer to call those delegating methods as `do..`.
Therefore `doAddMessagesToGroup()`.
Plus I believe it must not be `public` here.